### ISSUE TITLE
EP2 gets abnormal hot

### ISSUE THREAD

**Issue by E1yot** [FormationFlight/FormationFlight#54]
I flashed V5.0.0-rc1 to Happymodell EP2 TCXO. The first one gets so hot, what the tower antenna fall down and some other parts moved on the pcb.
I ordered new RX, tested with ELRS and they worked normal. After the test I flashed FF V5.0.0-rc1. On a short benchtest without FC, everything works. The webinterface shows correct status. In the modell, also everything seems to go well, FC and iNav were detected, but the RX gets hot. Since I monitored the temperature, I could switch off, before any damage occurs.

**To Reproduce**
1. Flash the EP2 TCXO with the right firmware and connect it to the FC
2. Check the temperature, it gets hot in short time

**Expected behavior**
Temperature should be in the range like in oparation as ELRS RX

**Observed behavior**
Temperature rises over limits within short time

**FC Firmware / Version**
iNav 6.1.1, on AtomRC F405 NAVI

**Additional context**
I think, the voltage regulator is overloaded, since WiFi is activ and connected. Possible solution would be to disable the SX1280 with WiFi connected and disable WiFi while armed. Also power for WiFi should be limited.

**Comment by MUSTARDTIGERFPV** [comment]
Hey @E1yot! I've run EP2s for all my aircraft and in testing on the bench for hours at a time. They get warm, but never enough to desolder parts! What voltage are you feeding the EP2? Is it possible you're running it off a servo rail at 6V or 7V?

**Comment by pitts-mo** [comment]
Depending on how many people have this issue it might be worth looking into. I think this issue requires an interested/affected party to put in some research/testing. 

more info:
WiFi on my early HM EP2 RXs (not TCXO) is no longer usable when flashed with Formation Flight but they were fine with ELRS. It seemed like the whole RX was resetting and dropping WiFi connection shortly after attempt connections. I suspected my issue was related to the poor voltage regulator spec issue from the early batches.

**Comment by E1yot** [comment]
Voltage is 5.090 V, measured with a Fluke 179 mulitmeter.

There is nearly zero area for cooling. Current consumption of the ESP while WIFI transmitting data is 170 mA (datasheet value). With a voltage drop of 1.7 V (3.3V output), the LDO has a power dissipation of arround 300 mW. Without cooling it is a lot.

@MUSTARDTIGERFPV Do you have WiFi connection and webinterface activ at your bench tests?

**Comment by MUSTARDTIGERFPV** [comment]
@E1yot

I tried to reproduce this. I flashed a brand-new EP2 with FormationFlight 5.0.0 and fed it ~4.97V as measured by my oscilloscope. I let it run for ~30 minutes sitting on a bench with no airflow, and it's still comfortably warm to the touch.

It's drawing 70mA on average as measured by a 1 ohm shunt resistor measured by oscilloscope, including the ~1ms peaks when transmissions occur which are admittedly quite high (400mA). Even so, it's fully within reason for temperature.

If your PCB got hot enough for the solder to melt, I'd venture to guess something else is wrong. Are you able to measure the average current draw with your multimeter or something along those lines?

BTW we can have this conversion with much less latency in the FormationFlight Discord: https://discord.gg/Jp8Pv4b9
